modules/lvgl/Kconfig

```diff
@@ -12,6 +12,10 @@ config LVGL
 
 if LVGL
 
+config LV_Z_SYS_INIT_PRIORITY
+	int
+	default 90
```

Comment:
Add a prompt to allow for the user to modify the value?

Reply:
True, added this. Also for the other priority defined by the module.